leshan-server-core/src/main/java/org/eclipse/leshan/server/registration/Registration.java

```diff
@@ -74,7 +75,58 @@
     private final String rootPath;
 
     private final Date lastUpdate;
+    
+    /** The LwM2m Queue Mode management object */
+    private LwM2mQueue queue;
```

Comment:
Registration is immutable. This is a choice to better handle persistence and threadsafeness.
And as discussed in #265  I don't think presence state should be store in registration because we don't want to persist it or to share it between server instance(for cluster)